core/QuickForm2.php

```diff
@@ -80,24 +80,6 @@ public function addElement(
         return parent::addElement($elementOrType, $name, $attributes, $data);
     }
 
-    public function setChecked($nameElement)
-    {
-        foreach ($this->_elements as $key => $value) {
```

Comment:
The methods are not used anywhere. Also `$this->_elements` does not even exits in this class and would fail anyway.